static/index.js

```diff
@@ -103,6 +103,19 @@ var setupCoffeeCache = function(cacheDir) {
   CoffeeCache.register();
 }
 
+var isPortableMode = function() {
+  // No portable mode on non-Windows
+  if (process.platform !== 'win32') return false;
+  
+  // DevMode? Nope
+  var devMode = loadSettings.devMode || !loadSettings.resourcePath.startsWith(process.resourcesPath + path.sep);
+  if (devMode) return false;
+  
+  // Compare our EXE's path to where it would normally be in an installed app
+  var ourPath = process.execPath.toLowerCase();
+  return (ourPath.indexOf(process.env.USERPROFILE.toLowerCase()) === 0);
```

Comment:
This is a typo that I fixed but didn't push apparently, this should be `LOCALAPPDATA`, which makes *way* more sense :)